--- /usr/local/lib/python2.7/dist-packages/mapproxy/util/yaml.py        2015-07-28 10:44:12.788000000 +0000
+++ /tmp/yaml.py        2015-07-28 10:44:06.796000000 +0000
@@ -35,13 +35,13 @@
     Load yaml from file object or string.
     """
     try:
-        if getattr(yaml, '__with_libyaml__', False):
-            try:
-                return yaml.load(doc, Loader=yaml.CLoader)
-            except AttributeError:
-                # handle cases where __with_libyaml__ is True but
-                # CLoader doesn't work (missing .dispose())
-                return yaml.load(doc)
+        #if getattr(yaml, '__with_libyaml__', False):
+        #    try:
+        #        return yaml.load(doc, Loader=yaml.CLoader)
+        #    except AttributeError:
+        #        # handle cases where __with_libyaml__ is True but
+        #        # CLoader doesn't work (missing .dispose())
+        #        return yaml.load(doc)
         return yaml.load(doc)
     except (yaml.scanner.ScannerError, yaml.parser.ParserError) as ex:
         raise YAMLError(str(ex))

